packages/dnote/build.sh

```diff
@@ -1,37 +1,24 @@
 TERMUX_PKG_HOMEPAGE=https://www.getdnote.com/
 TERMUX_PKG_DESCRIPTION="A simple command line notebook for programmers"
 TERMUX_PKG_LICENSE="AGPL-V3,GPL-3.0"
-TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
-TERMUX_PKG_VERSION=2.0.1
-TERMUX_PKG_REVISION=2
-TERMUX_PKG_SRCURL=https://github.com/dnote/dnote/archive/refs/tags/server-v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=28cc3bf93b3849b9a4a5e65e531f8a34d6be6048427d924c56ab8c7887676bad
+TERMUX_PKG_MAINTAINER="Ravener <ravener.anime@gmail.com>"
+TERMUX_PKG_VERSION=0.12.0
```

Comment:
Need to set an epoch here since version is changed from 2.0.1 to 0.12.0:

```
TERMUX_PKG_VERSION=1:0.12.0
```

and SRCURL then needs to be updated to use `${TERMUX_PKG_VERSION:2}`